packages/web-components/fast-components/src/tabs/fixtures/base.html

```diff
@@ -91,7 +106,7 @@ <h4>Icons - Vertical</h4>
             Tab three content. This is for testing.
         </fast-tab-panel>
     </fast-tabs>
-    <h4>Suplemental content</h4>
+    <h4>Supplemental content</h4>
```

Comment:
This should be fixed in fast-components-msft also

Reply:
ahh good catch